src/core/drive/preloader.ts

```diff
@@ -28,23 +27,25 @@ export class Preloader {
     const links = element.querySelectorAll('a[data-turbo-preload="load"]')
```

Comment:
Does executing this query with additional type information help avoid the `instances` check?

```js
for (const link of element.querySelectorAll<HTMLAnchorElement>('a[data-turbo-preload="load"]')) {
  this.preloadURL(link)
}
```

Reply:
Resolved in https://github.com/hotwired/turbo/pull/552/commits/15fedf0af59c68e77325c5c60bf45949437d73ac